test/integration/pipeline/tx_pipeline_integration_test.cpp

```diff
@@ -187,6 +187,7 @@ TEST(PipelineIntegrationTest, SendQuery) {
  * @given some user
  * @when sending sample AddAssetQuantity transaction to the ledger
  * @then receive STATELESS_VALIDATION_SUCCESS status on that tx
+ * @then wait for proposal and block
```

Comment:
Multiple then here, use just `and` instead

Reply:
Please use and as @l4l suggested